File: Server/class/food.js
var shortID = require("shortid");
var vector2 = require("./vector2.js");
var vector3 = require("./vector3.js");
var color = require("./color.js");
module.exports = class food {
	constructor() {
		this.id = shortID.generate();
		this.position = new vector2();
		this.scale = new vector3();
		this.color = new color();
	}
}

File: scripts/node/Food.cs
[System.Serializable]
public class Food 
{

    public NodePosition position = new NodePosition();
    public NodeScale scale = new NodeScale();
    public PlayerColor color = new PlayerColor();
    public string id = "";

    public Food(string ID, NodePosition pos, NodeScale s, PlayerColor c)
    {
        id = ID;
        position = pos;
        scale = s;
        color = c;
    }
}
